### ISSUE TITLE
Fails to build under Linux.

### ISSUE THREAD

**Issue by jasonjgw** [NSoiffer/MathCATForPython#16]
The Python module fails to build under Linux:

* The architecture is set in .cargo/config.toml to 32-bit Windows x86.
* Once this file is removed, the code in `build.rs` fails, since the "Rules" directory does not exist, hence cannot be deleted.

Would it be better to separate out the Python module completely from the NVDA-specific code (e.g., one repository for the Python module and another for the NVDA add-on)? Then a Linux package would simply distribute the Python module.

If not, what other approach should be taken to get a platform-independent Python module that isn't bound up with Windows/NVDA?

This isn't urgent, of course; it won't matter until someone starts to develop Orca support. I am creating this issue for tracking purposes.

**Comment by NSoiffer** [comment]
I thought I fixed this, but it was only a partial fix. I moved the "target" part to the shell script `build-addon.sh` but didn't remove the .cargo dir.

I removed the "rmdir" from `build.rs` and put it in the shell script. I renamed the script to `build-nvda-addon.sh` to make its purpose clearer.

`location` in `buid.rs` probably should be put inside a `cfg`, but since I don't know what it should be for ORCA, I'm leaving that to later.

Let me know if there are still problems.
